### PR TITLE
Fix Alpine Links and update to .NET Core SDK 2.1.400

### PR DESCRIPTION
**Pull request by JRAlexander**
Fix Alpine Links and update to .NET Core SDK 2.1.400

Fixes https://github.com/dotnet/docs/issues/7103

### CODE REVIEW

File: docs/core/linux-prerequisites.md [hunk 1]
```diff
@@ -45,9 +45,9 @@ NET Core 2.0 is supported on the following Linux 64-bit (`x86_64` or `amd64`) di
 * Red Hat Enterprise Linux 7, 6
 * CentOS 7
 * Oracle Linux 7
-* Fedora 27
+* Fedora 26,25
 * Debian 9, 8.7 or later versions
-* Ubuntu 18.04, 16.04, 14.04
+* Ubuntu 18.04, 17.10, 16.04, 14.04
```
Comment (mairaw) [inline docs/core/linux-prerequisites.md]: same. list on that page lists 18.04, 16.04, 14.04

File: docs/core/linux-prerequisites.md [hunk 2]
```diff
@@ -581,12 +587,13 @@ To install .NET Core 2.x for supported SUSE Linux Enterprise Server and OpenSUSE
 
 Download and follow the .NET Core 2.1 installation instructions for supported Alpine Linux versions (64 bit) at the following links:
 
-* .NET Core Runtime 2.1.2 [download link](https://www.microsoft.com/net/download/linux-package-manager/sles/runtime-2.1.2)
-* .NET Core SDK 2.1.302 [download link](https://www.microsoft.com/net/download/linux-package-manager/sles/sdk-2.1.302)
-* .NET Core Runtime 2.1.1 [download link](https://www.microsoft.com/net/download/linux-package-manager/sles/runtime-2.1.1)
-* .NET Core SDK 2.1.301 [download link](https://www.microsoft.com/net/download/linux-package-manager/sles/sdk-2.1.301)
-* .NET Core Runtime 2.1.0 [download link](https://www.microsoft.com/net/download/linux-package-manager/sles/runtime-2.1.0)
-* .NET Core SDK 2.1.300 [download link](https://www.microsoft.com/net/download/linux-package-manager/sles/sdk-2.1.300)
+* .NET Core Runtime 2.1.2 [download link](https://www.microsoft.com/net/download/thank-you/dotnet-runtime-2.1.2-linux-x64-alpine-binaries)
```
Comment (mairaw) [inline docs/core/linux-prerequisites.md]: @leecow we don't have a similar page for alpine like the other distributions have?

**Review by mairaw — APPROVED** [review]
LGTM but had some questions about some of the changes

File: docs/core/linux-prerequisites.md [hunk 3]
```diff
@@ -45,9 +45,9 @@ NET Core 2.0 is supported on the following Linux 64-bit (`x86_64` or `amd64`) di
 * Red Hat Enterprise Linux 7, 6
 * CentOS 7
 * Oracle Linux 7
-* Fedora 27
+* Fedora 26,25
```
[the remark this answers is not in the document]
Comment (JRAlexander) [reply docs/core/linux-prerequisites.md]: 26 and 25 are the downloadable versions.

File: docs/core/linux-prerequisites.md (hunk 2 above)
Comment (JRAlexander) [reply docs/core/linux-prerequisites.md]: Nope

File: docs/core/linux-prerequisites.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (mairaw) [reply docs/core/linux-prerequisites.md]: I don't follow. 25 and 26 are EOL already.

File: docs/core/linux-prerequisites.md (hunk 2 above)
Comment (leecow) [reply docs/core/linux-prerequisites.md]: That's correct. At the moment it's only referenced in the 'Other downloads' table at the bottom of https://www.microsoft.com/net/download, under the Linux tab.

File: docs/core/linux-prerequisites.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (JRAlexander) [reply docs/core/linux-prerequisites.md]: Changed

**Pull request merged**